sklearn/utils/tests/test_validation.py

```diff
@@ -19,12 +20,12 @@
 from sklearn.svm import SVR
 
 from sklearn.datasets import make_blobs
-from sklearn.utils import as_float_array, check_array
-from sklearn.utils.estimator_checks import NotAnArray
 from sklearn.utils.validation import (
-        NotFittedError,
-        has_fit_parameter,
-        check_is_fitted)
+    NotFittedError,
+    has_fit_parameter,
+    check_is_fitted)
```

Comment:
I agree. I did not intend to change those import lines as part of this PR but apparently my editor fixed the indenting without asking me :)